src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -306,22 +345,29 @@ private void VisitArrayCommon<T>(ImmutableArray<T> list, string header, bool log
 
         internal void VisitSymbolArrayElement(ISymbol element)
         {
+            Indent();
```

Comment:
:bulb: The indentation change should be implemented as its own separate PR against the master branch, after the other test changes are completed and merged.